generate-docs/tools/dts-diff.ts

```diff
@@ -0,0 +1,31 @@
+import * as fsx from "fs-extra";
+import { diff } from 'deep-diff';
+import { APISet, parseDTS } from './dts-utilities';
+
+tryCatch(async () => {
+    //fsx.writeFileSync("./tool-inputs/dt-release.d.ts", await fetchAndThrowOnError("https://raw.githubusercontent.com/DefinitelyTyped/DefinitelyTyped/master/types/office-js/index.d.ts", "text"));
```

Comment:
just curious about all the commented out stuff...

Reply:
That was a separate tool I'm working on. Didn't mean to include that in the PR.